packages/metals-vscode/package.json

```diff
@@ -484,6 +484,11 @@
         "category": "Metals",
         "title": "Zip reports"
       },
+      {
+        "command": "metals.reset-workspace",
+        "category": "Metals",
+        "title": "Reset workspace"
```

Comment:
nitpick: is `Reset workspace` descriptive enough for people? Maybe sth more similar to what Intellij has will be better? Maybe sth more verbose like "Clear caches and reimport project"?

Reply:
Good idea, I changed it